apps/site/src/pages/_app.page.tsx

```diff
@@ -60,7 +60,19 @@ const MyApp = ({
   const [user, setUser] = useState<UserState>("loading");
 
   const refetchUser = useCallback(async () => {
-    const { data, error } = await apiClient.get<ApiMeResponse>("me");
+    const { data, error } = await apiClient.get<ApiMeResponse>("me", {
+      "axios-retry": {
+        onRetry: (retryCount, axiosError) => {
+          Sentry.captureMessage("Retrying /api/me", {
+            extra: { retryCount, axiosError },
+          });
+        },
+        retries: 2,
+        retryCondition: (axiosError) => {
+          return axiosError.response?.status !== 200;
+        },
+      },
+    });
```

Comment:
We can move this logic into `apiClient` → `get` later but I’d limit the scope of retries for now just in case.